src/CodeGen_ARM.cpp

```diff
@@ -885,6 +883,9 @@ void CodeGen_ARM::visit(const Load *op) {
             return;
         }
 
+#if LLVM_VERSION < 37
```

Comment:
I think we've already stopped supporting 36 and below, so maybe we should just delete this code?